config/crd/bases/secrets.hashicorp.com_vaultdynamicsecrets.yaml

```diff
@@ -75,6 +75,13 @@ spec:
               namespace:
                 description: Namespace where the secrets engine is mounted in Vault.
                 type: string
+              params:
+                additionalProperties:
+                  type: string
+                description: Params that can be passed when requesting credentials/secrets.
+                  Please consult https://developer.hashicorp.com/vault/docs/secrets
+                  if one is uncertain about what 'params' should/can be set to.
```

Comment:
Perhaps worth adding that it implies a `POST`? As HTTP verbs are the terms in which our API is written.

Reply:
Fixed in https://github.com/hashicorp/vault-secrets-operator/commit/500f1e0f926cac8106c76751f8316a083cf65d90